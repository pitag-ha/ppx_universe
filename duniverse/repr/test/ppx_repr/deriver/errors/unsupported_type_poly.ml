type t = { v : 'a. 'a } [@@deriving repr]
