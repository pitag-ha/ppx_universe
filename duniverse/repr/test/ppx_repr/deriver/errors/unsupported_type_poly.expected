File "unsupported_type_poly.ml", line 1, characters 15-21:
Error: ppx_repr: universally-quantified type 'a . 'a encountered. Irmin types must be grounded.
